catlin/pkg/cmd/validate/validate.go

```diff
@@ -142,10 +142,16 @@ func validate(cli app.CLI, path string) error {
 		return err
 	}
 
+	// Get the predefined category list
+	cat, err := validator.GetCategories()
```

Comment:
this will be executed for each file. `validate(cli app.CLI, path string)` is executed on each file.
https://github.com/tektoncd/plumbing/blob/3350cf6d8fbc769d0c8af369344b8263ad9c394b/catlin/pkg/cmd/validate/validate.go#L114